microsoft-edge/extensions-chromium/developer-guide/sidebar.md

```diff
@@ -0,0 +1,625 @@
+#### PanelBehavior
+
+<!-- compare https://developer.chrome.com/docs/extensions/reference/sidePanel/#type-PanelBehavior -->
+
+Panel behavior, including `openPanelOnActionClick`.
+
+<!-- ---------- -->
+**PROPERTIES**
+
+###### openPanelOnActionClick
+
+Whether clicking the extension's icon toggles showing the extension's entry in the sidebar.  The default is `false`.
+
+Type: `boolean`, optional
+
+
+<!-- ====================================================================== -->
+#### PanelOptions
+
+<!-- compare https://developer.chrome.com/docs/extensions/reference/sidePanel/#type-PanelOptions -->
+
+Panel options, including `enabled`, `path`, and `tabId`.
+
+<!-- ---------- -->
+**PROPERTIES**
+
+###### enabled
+
+Whether the sidebar is enabled.  The default is `true`.
+
+Type: `boolean`, optional
+
+
+###### path
+
+The path to the sidebar HTML file to use.  This must be a local resource within the extension package.
+
+Type: `string`, optional
+
+The default is a blank page with no UI.
+
+
+###### tabId
+
+Type: `number`, optional
+
+If a `tabID` is specified, the sidebar options (`enabled`, `path`, and `tabId`) only apply to the tab that has this ID.  If a `tabID` is not specified, these options set the default behavior, which is used for any tab that doesn't have specific settings defined.
+
+<!-- todo: what sentence do we want that's equivalent to this sentence?  from https://developer.chrome.com/docs/extensions/reference/sidePanel/#type-PanelOptions --
+If the same `path` is set for this `tabId` and the default `tabId`, then the panel for this `tabId` will be a different instance than the panel for the default `tabId`.
+
+our draft sentence:
+
+... the same instance ...
+-->
+
+
+<!-- ====================================================================== -->
+#### SidePanel
+
+<!-- compare https://developer.chrome.com/docs/extensions/reference/sidePanel/#type-SidePanel -->
+
+Sidebar properties, including `default_path`.
+
+<!-- ---------- -->
+**PROPERTIES**
+
+<!-- ---------- -->
+###### default_path
+
+Type: `string`, required
+
+The developer-specified path for sidebar display.
+
+
+<!-- ====================================================================== -->
+## Methods for sidebar extensions
+
+<!-- compare https://developer.chrome.com/docs/extensions/reference/sidePanel/#method (empty heading) -->
+
+* [getOptions](#getoptions)
+* [getPanelBehavior](#getpanelbehavior)
+* [setOptions](#setoptions)
+* [setPanelBehavior](#setpanelbehavior)
+
+
+<!-- ====================================================================== -->
+#### getOptions
+
+<!-- compare https://developer.chrome.com/docs/extensions/reference/sidePanel/#method-getOptions -->
+
+Returns the active panel configuration.
+
+
+<!-- ---------- -->
+**SYNTAX**
+
+```js
+chrome.sidePanel.getOptions(
+  options: GetPanelOptions,
+  callback?: function,
+)
+```
+
+* Promise - This method an return its result via a Promise, in Manifest V3 or later.
```

Comment:
The sidePanel API applies only to manifest v3, so the caveat is not needed. And it should be "can return its result"?

Reply:
todo: check wording in the "Promise" label in  https://developer.chrome.com/docs/extensions/reference/sidePanel/#method-getOptions
todo: change to (typo):
Promise - This method can return its result via a Promise.